src/environments/environment.ts

```diff
@@ -3,7 +3,9 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  apiKey: "39f872d65992fa9149e66aff8b03363f",
+  getCharacterUrl: "https://gateway.marvel.com:443/v1/public/characters/"
```

Comment:
only the base url should be in the environment file and not the whole url
i.e baseURL:'https://gateway.marvel.com:443/v1/public/'